NEWS.md

```diff
@@ -1,6 +1,11 @@
 # remotes (development version)
 
 * Another fix for the mixed binary and source dependency issue, it should hopefully be fully squashed now (#296)
+* Remotes functions can now install dependencies from additional DESCRIPTION
+  fields, e.g. passing `dependencies = "Config/pkgdown"` will install the
```

Comment:
Should this be `Config/remotes/...` to allow for use of `Config/` by other packages?

Reply:
Or, actually `Config/Needs/...` seems even better, as per https://github.com/r-lib/remotes/issues/459#issuecomment-590498663

Reply:
The extra namespacing seems unneeded until there is a clash, and if there ever is a clash then we can add just start using it, the code would stay the same.

Reply:
If we don't have an extra namespace, that means that we are claiming the whole `Config/` namespace for dependency purposes. E.g. if we install packages in `Config/pkgdown` when building the pkgdown site, that rules out for example pkgdown storing its own config in `Config/pkgdown`.

Reply:
Ok, I added a 'Config/needs/website' example in the examples

Reply:
Thanks!

Reply:
Given that we now know about it, I suspect the chances of using `Config` for other packages is high, and I think the most natural way to divide up the `Config` namespace is via package name (since that already provides a system for "claiming" a name)

Reply:
Oh, so you are saying `Config/remotes/website` is what this should be called.

My main hesitation is that I would guess in the future pak and renv may also use this field, but maybe that is ok.

Reply:
Also calling it `Config/remotes` may confuse people with the existing `Remotes:` field.

Reply:
Yeah, I think `Config/Needs` is more readable, and renv and pak will also use it. I don't think `Config/remotes` is bad, though, I think that's also reasonable.